ruby.html.markdown

```diff
@@ -145,6 +145,12 @@ x = y = 10 #=> 10
 x #=> 10
 y #=> 10
 
+# Ruby also has a or-equals operator
+x = 10 #=> 10
```

Comment:
That's just compound assignment at work. Could you show more generally how it works, adding `+=`, say?